src/main/java/emiya/emiyaexception/EmptyDeleteException.java

```diff
@@ -0,0 +1,9 @@
+package emiya.emiyaexception;
+
+public class EmptyDeleteException extends EmiyaException{

```

Comment:
Would be good to add javadocs to your exception classes too! There are quite a few of exception classes without javadocs headers